kubetest/util.go

```diff
@@ -140,8 +141,14 @@ func finishRunning(cmd *exec.Cmd) error {
 
 	finished := make(chan error)
 
+	sig_channel := make(chan os.Signal, 1)
+	signal.Notify(sig_channel, os.Interrupt)
+
 	go func() {
 		finished <- cmd.Wait()
+		sig := <-sig_channel
+		log.Printf("Killing %v after receiving %s", -cmd.Process.Pid, sig)
+		syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL)
```

Comment:
humm. If adding error handling here, "kubetest/util.go:151: result of fmt.Errorf call not used" error happens as we discussed yesterday with BenTheElder.
I guess that is the reason why current terminate.C case doesn't contain the error handling of syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL).
So it would be good enough to skip it at this time, I feel.